DALK.PL_ANALYZER/Views/Matches/Index.cshtml

```diff
@@ -3,41 +3,77 @@
 @{
     ViewBag.Title = "Meczyki";
 }
+ 
+    @{
+        string allMatchesClass = "layer";
+        if (Model.Count == 1)
+        {
+            allMatchesClass += " singleMatch";
+        } else {
+            allMatchesClass += " matches";
+        }
+    }
+    <div class="@allMatchesClass">
+        @foreach (DALK.PL_ANALYZER.Models.Matches.Match m in Model)
+        {
+            DALK.PL_ANALYZER.Models.Matches.PlayedMatch playedMatch = m as DALK.PL_ANALYZER.Models.Matches.PlayedMatch;
+            string matchClass = "match";
+            if (playedMatch == null)
+            { 
+                matchClass += "__notPlayed";
+            } 
+            <div class="@matchClass">
+                <div class="match__heading">
+                    <div class="match__gametitle">@m.GetGameTitle()</div>
+                    <div class="match__level">@m.League @m.Season, @m.DateTime.ToString("yyyy-MM-dd HH:mm:ss")</div>
+                </div>
+                @{
+                    if (playedMatch != null)
+                    {
+                        <div class="match__result">
+                            <div class="match__team match__winner">
+                                <img class="match__icon" src="@playedMatch.Winner.IconUrl">
+                                <div class="match__teamname">@playedMatch.Winner.ToString()</div>
```

Comment:
Maybe winner should be already a string? ViewModel properties maybe all should be already a strings so there is no need to do any code at view

Reply:
DONE.